packages/psss-server/src/routes/LogoutRoute.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Route } from './Route';
+
+export class LogoutRoute extends Route {
+  async buildResponse() {
+    await this.sessionModel.deleteById(this.sessionCookie.id);
+    this.sessionCookie?.reset();
```

Comment:
I'm not sure if this will work if `reset` is not defined, since it will try to invoke `undefined` as a function?

(I realise the existing code was like that, but it may be still worth checking)

Reply:
Yeah this is one of the ts errors I gave up on. I know that at the point `buildResponse` is called that `sessionCookie` will be defined on the class, but that's just because I know the logic of the parent class. It doesn't get defined in the constructor, so I can't make it mandatory...thought I would leave it for later!

Reply:
Good call!